server/scopeserver/dataserver/modules/gserver/GServer.py

```diff
@@ -578,13 +577,8 @@ def getMyLooms(self, request, context):
         return s_pb2.MyLoomsReply(myLooms=my_looms, update=update)
 
     def getUUID(self, request, context):
-        if SCope.app_mode:
-            with open(os.path.join(self.dfh.get_config_dir(), "Permanent_Session_IDs.txt"), "r") as fh:
-                newUUID = fh.readline().rstrip("\n")
-                logger.info(f"IP {request.ip} connected to SCope. Running in App mode. Passing UUID {newUUID}.")
-        else:
-            newUUID = str(uuid.uuid4())
-        if newUUID not in self.dfh.get_current_UUIDs().keys():
+        newUUID = str(uuid.uuid4())
+        if newUUID not in self.dfh.get_current_UUIDs():
```

Comment:
@KrisDavie what should happen if `newUUID` _is_ in the current UUIDs?

Reply:
Final decision: make this unconditional

Reply:
Will you make a new PR for this?